railties/test/application/configuration_test.rb

```diff
@@ -169,7 +169,7 @@ def change
       assert_equal Rails.application.routes_reloader, AppTemplate::Application.routes_reloader
     end
 
-    test "Rails::Application responds to paths" do
+    test "AppTemplate::Application responds to paths" do
```

Comment:
`Rails::Application` is right. It's just using `AppTemplate` as an instance of `Rails::Application`.